osu.Game/Utils/GeometryUtils.cs

```diff
@@ -218,5 +219,137 @@ private static IEnumerable<Vector2> enumerateStartAndEndPositions(IEnumerable<IH
 
                 return new[] { h.Position };
             });
+
+        #region welzl_helpers
```

Comment:
Please use normal English for region descriptions rather than camel case identifiers